contrib/ohi-release-notes/action.yml

```
@@ -10,9 +10,6 @@ inputs:
   fail-if-held:
     description: Fail if the held toggle is active. Defaults to `true`.
     default: 'true'
-  link-dependencies-dictionary:
-    description: Sets the link dependency dictionary.
-    default: ''
```

```suggestion
  link-dependencies-dictionary:
    description: Sets the link dependency dictionary.
    default: '.github/rt-dictionary.yml'
```
Even if it is an OHI, I would let users modify it.

@alvarocabanas posted this in an internal blog post and we expect people to start using this, and they might use our contrib action that is ready to be used.

---

I think that the scope of this action is for ohi, that would involve an extra complexity that I'm not sure that is needed at this point, it could be added in the future I think ( if needed) ,We could probably all benefit from the same dictionary at the end.

---

so finally done in [18c1d36](https://github.com/newrelic/release-toolkit/pull/152/commits/18c1d36223c4664ded6407f4c027b892c1c4d00b), it was actually less complex than I imagine, thanks for the sugestion.